srcpkgs/rock64-uboot/template

```diff
@@ -0,0 +1,45 @@
+# Template file for 'rock64-uboot'
+pkgname=rock64-uboot
+version=2020.07
+revision=1
+archs="aarch64*"
+wrksrc="u-boot-${version}"
+hostmakedepends="flex bc python3 swig python3-devel dtc"
+makedepends="atf-rk3328-bl31"
+short_desc="Das U-Boot for the Rock64 SBC"
+maintainer="Cameron Nemo <cnemo@tutanota.com>"
+license="GPL-2.0-or-later, BSD-3-Clause"
+homepage="https://www.denx.de/wiki/U-Boot/"
+distfiles="ftp://ftp.denx.de/pub/u-boot/u-boot-${version}.tar.bz2"
```

Comment:
I think we can fetch it via `https`?

Reply:
oh true, I will adjust that then. (and for #21199).

Reply:
done here, will update the pbp PR at a later date